packages/app/src/app/overmind/state.ts

```diff
@@ -144,4 +152,10 @@ export const state: State = {
     github: false,
   },
   isProcessingPayment: false,
+  features: {
+    // Fallback values for when the features endpoint is not available
+    loginWithApple: true,
```

Comment:
Shouldn't it init as false? I guess such features should be unavailable until the API confirms its state, otherwise the user might see a layout shift if the API returns `false`

Reply:
I was thinking about this, but I was more concerned with breaking something in prod. This call is super fast, so there shouldn't be any shift as far as I tested. But wanted to make sure that in case it fails (for whatever reason) the default options are available as a fallback